algorithms/indexing/indexer.py

```diff
@@ -948,29 +948,11 @@ def _debug_write_reciprocal_lattice_points_as_pdb(
                 xs.sites_mod_short()
                 f.write(xs.as_pdb_file())
 
-    def debug_write_ccp4_map(self, map_data, file_name):
-        from iotbx import ccp4_map
-
-        gridding_first = (0, 0, 0)
-        gridding_last = map_data.all()
-        labels = ["cctbx.miller.fft_map"]
-        ccp4_map.write_ccp4_map(
-            file_name=file_name,
-            unit_cell=self.fft_cell,
-            space_group=sgtbx.space_group("P1"),
-            gridding_first=gridding_first,
-            gridding_last=gridding_last,
-            map_data=map_data,
-            labels=flex.std_string(labels),
-        )
-
```

Comment:
This method isn't used anywhere and isn't tested (or documented, not that that's exceptional in DIALS).  It's also broken, with an unused import, so I suggest hacking it out and burning it.